docs/api/browser-window.md

```diff
@@ -61,6 +61,7 @@ It creates a new `BrowserWindow` with native properties as set by the `options`.
   key is pressed.
 * `enable-larger-than-screen` Boolean - Enable the window to be resized larger
   than screen.
+* `background-color` Integer - Window's background color.
```

Comment:
`background-color` takes ARGB representation (0xFFFFFFFF), should that enough or accept individual components and form `SkColor` from it ? Need to clarify documentation, what should be the correct type here ?